docs/metrics/index.md

```diff
@@ -12,6 +12,20 @@ Each ML use case requires different metrics. Using the right metrics is critical
 comparing model performance. In each metrics guide, you can learn about the metric with examples, its limitations and
 biases, and its intended uses.
 
+## Common Machine Learning Tasks
```

Comment:
Instead of making another section under Metrics Glossary page, what do you think of linking the task doc by writing up a line about CV and common CV tasks under the `Computer Vision` section or under `Common Metrics` section (but personally I think OD should belong to CV)?  @gordonhart do you have any preference on where these task docs should go?

Reply:
How about this?

<img width="304" alt="Screenshot 2024-04-02 at 9 44 44 AM" src="https://github.com/kolenaIO/kolena/assets/9648886/6f384b35-b15f-4124-8626-7364565cbab4">

```yaml
    - Computer Vision:
      - Computer Vision Tasks:
        - metrics/object-detection.md
      - metrics/geometry-matching.md
      - metrics/iou.md
```

While we're here, I'd also encourage moving `Mean Average Error` to the "Common Metrics" section to remove the single-entry "Regression" section.